earth_enterprise/src/fusion/autoingest/tools/SConscript

```diff
@@ -22,7 +22,7 @@ Import('env')
 # ***  apps that are NOT installed
 # ****************************************************************************
 env.executable('gefindsrconvol', ['gefindsrconvol.cpp'],
-               LIBS=['geautoingest', 'gecommon', 'gemiscconfig', 'geutil'])
+               LIBS=['geautoingest', 'gecommon', 'gemiscconfig', 'geutil', 'qt-mt'])
```

Comment:
Any reason for this change? This doesn't seem related to the cache stuff.

Reply:
This was added due to some compile errors I encountered when calculating the size of QStrings in CacheSizeCalculations.